### PR TITLE
Milestone2

### PR DESCRIPTION
**Pull request by Abdona**
The Newsweeks Clone page.
Technologies used in this project :

- Html
- CSS: mapped from the SCSS file.
- SCSS: I created some new rules to add specialized style to the project
- Bootstrap5

### CODE REVIEW

**Review by peterrobert — CHANGES_REQUESTED** [review]
### INVALID 🛑 
Hello, @Abdona great effort submitting your project for a review. Good work 👏🏿 . However, for your project to be reviewed, it has to be professional. It has to meet the basic requirements. Kindly check below: ⬇️ 

- [ ] Kindly add a descriptive pull request message. This will make your project professional. Kindly add a title and short bullet points explaining what you have worked on this milestone. Kindly use [this article](https://microverse.zendesk.com/hc/en-us/articles/360055660814-Professional-GitHub-repository)

But no worries. Just create a descriptive pull request message and submit your project for another review.

### if you have any question reach me via slack @peterrobert

### HAPPY CODING 💻

**Review by jubaan — CHANGES_REQUESTED** [review]
[![Microverse][microverse-shield]][microverse-url]
<div>
<h1 align="center"> ⚠️ 🛠️ ⚠️ 🛠️ Changes Required 🛠️ ⚠️ 🛠️ ⚠️ </h1>
<br />
<p><b>"Fail fast, fail often"</b> and don't let this discourage you, because <b>only by failing do we learn to succeed</b>.</p>
<details open="open">
  <summary>Review Summary</summary>
  <ol>
    <li><a href="#congrats"> 💯 Awesome work 👏🏼 👏🏼 </a></li>
    <li><a href="#mandatory"> ⚠️ Things to Improve </a> 🚨 🚨 </li>
    <li><a href="#optional"> 💡Nice to have and other comments ✍🏼 ✍🏼 </a></li>
    <li><a href="#contact"> 🧔🏼 About your Reviewer </a> (you don't know it... but we are just automated bots 🤖) </li>
    <li><a href="#help"> 🆘 <b> 😭 Feeling stuck? or Need help</b>? 🙏🏼 🙏🏼 </a></li>
  </ol>
</details>
</div>

<h3 id="congrats"> 1. :100: Awesome work</h3>
<details close="close">
<summary> [click to open] Big or little achievements, but achievements at the end. <b>Feel proud that you're learning and improving everyday.</b></summary>



</details>

<hr>
<h3 id="mandatory"> 2. ⚠️ Things to Improve (or the To-Do list to get the project approved) </h3> 
<details close="close">
<summary> [click to open] Working on these observations will help you advance and get closer to the goal 🎯, postpone them or do them half-ways and they'll haunt you 👻 👻... FOREVER!! (It will feel like that, trust me). </summary>


- [ ] More stories and Opinion elements are not displaying like the original desing in desktop view
image
- [ ] Sections should be displayed in this order on mobile view
image
- [ ] Remove the `!important` from the SCSS file. 




</details>
<p><b>NOTE</b>This list is or could be divided in what is show in this comment and inline comments posted below.<p>
<hr>

<h3 id="optional"> 3. 💡 Nice to have and other comments </h3>
<details close="close">
<summary> [click to open] </h3></summary>


- [x] Nothing more to add.


</details>

<h3 id="contact"> 4. 🧔🏼 About your Reviewer </h3>
<details close="close">
<summary> [click to open] Say "hello world!" on Slack: @jubaan</summary>
<br />
<div align="center">
  <a src="https://www.github.com/jubaan">
    Julio Añoveros
  </a>
  <p>
    Julio Añoveros | @jubaan</br></br>
    <b>Full-stack Web Developer</b> and</br>
    🤖 Code Reviewer at <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a>:rocket:
  </p>
</div>
</details>

<h3 id="help"> 5. 🆘 <b> 😭 Feeling stuck? or Need help</b>? 🙏🏼 🙏🏼</h3>
<details close="close">
  <summary> [click to open] <b>Feel stuck?</b> </summary>
  <br>
  <p> First breath... just do it. Take 15-20 min away from the computer, stretch, meditate, do some exercise, drink water, or do whatever is necessary to distract your mind from the obstacle.</p>
  <p><b>Sometimes you just need to get away to see things with a new perspective.</b></p>
</details>

<details close="close">
  <summary> [click to open] <b>Need help?</b> </summary>
  <br>
  <p> Send me a DM in <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a> Students Slack, you'll find me as @jubaan. If I'm not available there are other Code Reviewers you can contact. They'll help as well.</p>
  <p>Remember that <a href="https://www.microverse.org/?grsf=kfm2ej">Microverse</a> has other channels where you can get an answer to your questions:</p>
  <ol>
    <li>The <b>students channel 🧑‍🎓 </b> in slack,</li>
    <li>your <b>teammates 👥 </b>,</li>
    <li>your <b>mentor 👨🏼 👩🏼 </b>, and</li>
    <li><b>Student Success 🏫 </b>.</li>
  </ol>
  <p><b> DON'T hesitate to ask! </b></p>
</details>
<p>If all of the above doesn't help... get a <a href="https://rubberduckdebugging.com/">rubber duck</a> ... it would be great to own one 😄. </p>

<h2 align="center"> Keep up the great work :muscle: and Happy Coding :smiley: :computer:!! </h1>

[microverse-shield]: https://img.shields.io/badge/-microverse-%236F23FF?style=for-the-badge
[microverse-url]: https://www.microverse.org/?grsf=kfm2ej
[bootstrap-shield]: https://img.shields.io/badge/Bootstrap-white?style=for-the-badge&logo=bootstrap
[bootstrap-url]: https://www.microverse.org/?grsf=kfm2ej

**Review by mikenath223 — CHANGES_REQUESTED** [review]
### Hello @Abdona  👋

**You've done pretty well here fixing a few issues raised in the previous review. 👌**

However, let's make a couple of changes.

- [x] You seemed to have forgotten to add linter config files to the project, kindly add them as they help to improve the professionalism of the project. Here is the [link](https://github.com/microverseinc/linters-config/tree/master/html-css)

- Please refer to other comments on the code.

### **STATUS: CHANGES REQUESTED** 👷‍♂️

Kindly fix these issues and request another review👍
Feel free to contact me on slack(michgolden) if you need further clarification.

**HAPPY CODING 🎉💻🎉**

**Michgolden Ukeje (CR)**

File: Assests/scss/main.css [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+}
+
+.border-new {
+  border: 1px solid #e0e1e1;
+}
+
+@media (max-width: 767px) {
+  .p-xs {
+    padding: 0 !important;
+  }
```
Comment (mikenath223) [inline Assests/scss/main.css]: - [x] Kindly remove these `important` tags, there is also one below the code block, please remove it also. It's bad practice to use important tags to override previous styles.

**Review by rukundoeric — APPROVED** [review]
# Status: 🍾 PROJECT APPROVED 🍾
_Excellent job on this project guys. 🎆 Keep it up._

- ### To Highlight
  - [x] The project is clean. 💯
  - [x] The project follows Git flow. 💯

- ### To Improve
  - N/A

_Congratulations on the splendid job **@team** 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_

### _Keep working hard._
### _Happy Coding😃_
-----------------------------------------------------------------------------------------------------------------------------------------
**Regards,
[Rukundo Eric](https://rukundo-eric.herokuapp.com/) ,
[LinkedIn](https://www.linkedin.com/in/rukundo-eric/),
Slack: Rukundo Eric**

**Pull request closed without merging**